Makefile

```diff
@@ -76,5 +72,7 @@ clean:
 	@rm -rf datarobot_batch_scoring.egg-info build/* dist/* 
 	@rm -rf htmlcov
 	@rm -rf .coverage
+	@rm -rf .cache
 	@rm -f batch_scoring.spec batch_scoring_sse.spec batch_scoring_deployment_aware.spec
+	@rm -f *.shelve.*
```

Comment:
Just curious what is in there?

Reply:
As far as I understand it is sort of like Pickle files being made by the module during runtime - it was already being git-ignored but not removed during cleaning, if I understand it correctly. https://docs.python.org/3/library/shelve.html